data-processing/entities/sentences/types.py

```diff
@@ -6,3 +6,14 @@
 @dataclass(frozen=True)
 class Sentence(SerializableEntity):
     text: str
+    cleaned_text: str
```

Comment:
Why not just use the value of 'cleaned_text' for 'text'? I don't think anyone will plan to use the old 'text' field once we have a more cleaned text field.

Reply:
Sure! I deleted `text`.